sdgym/datasets.py

```diff
@@ -14,20 +14,22 @@
 LOGGER = logging.getLogger(__name__)
 
 DATASETS_PATH = Path(appdirs.user_data_dir()) / 'SDGym' / 'datasets'
-BUCKET = 'sdv-demo-datasets'
+BUCKET = 's3://sdv-demo-datasets'
 BUCKET_URL = 'https://{}.s3.amazonaws.com/'
 TIMESERIES_FIELDS = ['sequence_index', 'entity_columns', 'context_columns', 'deepecho_version']
 MODALITIES = ['single_table', 'multi_table', 'sequential']
+S3_PREFIX = 's3://'
 
 
 def download_dataset(modality, dataset_name, datasets_path=None, bucket=None, aws_key=None,
                      aws_secret=None):
     datasets_path = datasets_path or DATASETS_PATH / dataset_name
     bucket = bucket or BUCKET
+    bucket_name = bucket[len(S3_PREFIX):] if bucket.startswith(S3_PREFIX) else bucket
```

Comment:
minor but we can make this a function since you use it later too. Like a `get_bucket_name_from_path` of something